specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -1,2636 +1,2978 @@
+                        "type": "string",
+                        "maxLength": 64
+                    },
+                    {
+                        "name": "top",
+                        "in": "query",
+                        "description": "The number of person groups to list.",
+                        "type": "integer",
+                        "minimum": 1,
+                        "maximum": 1000,
+                        "default": 1000
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "A successful call returns an array of person groups and their information.",
+                        "schema": {
+                            "$ref": "#/definitions/PersonGroupResults"
+                        },
+                        "examples": {
+                            "application/json": [
+                                {
+                                    "personGroupId": "sample_group",
+                                    "name": "group1",
+                                    "userData": "User-provideddataattachedtothepersongroup"
+                                },
+                                {
+                                    "personGroupId": "sample_group2",
+                                    "name": "group2",
+                                    "userData": "User-provideddataattachedtothepersongroup"
+                                }
+                            ]
+                        }
+                    },
+                    "default": {
+                        "description": "Error response.",
+                        "schema": {
+                            "$ref": "#/definitions/APIError"
+                        }
+                    }
+                },
+                "produces": [
+                    "application/json"
+                ]
+            }
+        },
+        "/persongroups/{personGroupId}/train": {
+            "post": {
+                "description": "Queue a person group training task, the training task may not be started immediately.",
+                "operationId": "PersonGroup_Train",
+                "parameters": [
+                    {
+                        "name": "personGroupId",
+                        "in": "path",
+                        "description": "Target person group to be trained.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    }
+                ],
+                "responses": {
+                    "202": {
+                        "description": "The training task was queued successfully."
+                    },
+  
```

Comment:
Not the biggest fan of this name (could be somewhat ambiguous or confused with `VerifyResult`).